.github/workflows/labeler.yml

```diff
@@ -1,7 +1,21 @@
 name: Labeler
 
 on:
-- pull_request_target
+  # We need pull_request_target to be able to add labels to PRs from forks.
+  # Only allow pull_request_target when targeting main, not some historical branch.
+  #
+  # Make sure to don't introduce explicit checking out and installing/running
+  # untrusted user code into this workflow!
+  pull_request_target:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
```

Comment:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

> By default, a workflow only runs when a pull_request_target event's activity type is opened, synchronize, or reopened

Are you sure that we need to add `labeled`, `unlabeled` here?

Reply:
Let's remove.